src/Avalonia.Diagnostics/Diagnostics/ViewModels/HotKeyPageViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.ObjectModel;
+
+namespace Avalonia.Diagnostics.ViewModels;
+
+internal record HotKeyDescription(string Description, string LongDescription, string Gesture);
+internal class HotKeyPageViewModel : ViewModelBase
+{
+    public ObservableCollection<HotKeyDescription> HotKeys { get; } = new();
+
+    public HotKeyPageViewModel()
+    {
+        HotKeys = new()
+        {
+            new("Enable Snapshot Frames", "Pauses refreshing the Value Frames inspector for the selected Control", "Alt+S"),
+            new("Disable Snapshot Frames", "Resumes refreshing the Value Frames inspector for the selected Control", "Alt+D"),
+            new("Inspect Control Under Pointer", "Inspects the hovered Control in the Logical or Visual Tree Page", "Ctrl+Shift"),
+            new("Toggle Popup Freeze", "Prevents visible Popups from closing so they can be inspected", "Ctrl+Alt+F"),
+            new("Screenshot Selected Control", "Saves a Screenshot of the Selected Control in the Logical or Visual Tree Page", "F8")
```

Comment:
This PR looks good to me, but it would be a nice improvement to define KeyGestures in one place, and reuse in event handling and these descriptions.

Reply:
I'll amend to lift these into `DevToolsOptions` with some wiring work. It won't support mutability at runtime, but devs could provide their own KeyGestures at attach-time, read from their own config if they really wanted to.

Reply:
I didn't mean configurability, but only a static class with pre-defined key gestures. At least for now.
Long term, these should be mutable via devtools options, yeah.